textext/base.py

```diff
@@ -569,12 +569,15 @@ def parse_pdf_log(self):
         """
         with logger.debug("Parsing LaTeX log file"):
             from .texoutparse import LatexLogParser
-
             parser = LatexLogParser()
-            with open(self.tmp('log'), encoding='utf8') as f:
-                parser.process(f)
 
-            return parser.errors[0]
+            try:
+                with open(self.tmp('log'), encoding='utf8') as f:
+                    parser.process(f)
+                return parser.errors[0] if parser.errors else ""
+            except Exception as excpt:
+                return "TeX compilation failed and additionally parsing of TeX log file also failed ({}). \
+                        See output in stdout for more details.".format(type(excpt).__name__)
```

Comment:
Summary of my comments: 
```suggestion
            try:
                with open(self.tmp('log'), encoding='utf8') as f:
                    parser.process(f)
                return parser.errors[0]
            except Exception as ignored:
                return "TeX compilation failed. See stdout output for more details"
```